06.bowling_object/game.rb

```diff
@@ -0,0 +1,61 @@
+# frozen_string_literal: true
+
+require 'optparse'
+require './frame'
+
+class Game
+  def initialize(shot)
+    @shot = shot
+  end
+
+  def shots
+    @shot.split(',')
+  end
+
+  def main
+    @frames = frames
+    total = 0
+    (0..8).each do |n|
+      frame = Frame.new(@frames[n])
+      total += frame.scores
+      total += add_bonus(frame, n)
+    end
+    total + @frames.last.sum
+  end
+
+  def frames
+    scores = shots.map { |shot| Shot.new(shot).score }
+    frame = []
+    frames = []
+    scores.each do |score|
```

Comment:
```suggestion
    scores.each do |score|
      sleep 0.5
```

みたいにスリープ処理を呼ぶとめちゃくちゃ遅くなりませんか？
ロジック的に非効率な部分があるのでプログラム全体を見直してみてください。